code/controllers/master.dm

```diff
@@ -127,6 +128,12 @@ var/global/datum/controller/master/Master = new()
 
 				var/ran_subsystems = 0
 				for(var/datum/subsystem/SS in subsystems)
+					if(world.cpu >= 100)
+						//if world.cpu gets above 120,
+						//byond will pause most client updates for (about) 1.6 seconds.
+						//(1.6 seconds worth of ticks)
+						//We just stop running subsystems to avoid that.
+						break;
```

Comment:
Doing that is habit from php/c#/c++, I don't know why, but i always do that with continue and break but nothing else.